javascript/packages/orchestrator/src/providers/client.ts

```diff
@@ -56,6 +56,7 @@ export abstract class Client {
   abstract startPortForwarding(
     port: number,
     identifier: string,
+    namespace?: string,
```

Comment:
If it's available on the class, maybe not pass it as parameter ?

Reply:
There is a case where we use another `namespace`, from tracing services. That why is allowed to pass the namespace. I will leave for now, but we should clean that in the sdk.
Thanks!!